src/test/java/io/jenkins/plugins/opentelemetry/api/ReconfigurableOpenTelemetryTest.java

```diff
@@ -45,6 +46,7 @@ void test_configuration_through_system_properties() {
         try {
             Map<String, String> otelConfig = new HashMap<>();
             otelConfig.put("otel.exporter.otlp.endpoint", "http://localhost:4317");
+            otelConfig.put("otel.traces.exporter", "otlp");
```

Comment:
Shouldn't we create additional test cases rather than modifying those that are legit?

Reply:
They are not legit anymore. If there are no `exported` configurations, it does not make sense to enable the OpenTelemetry report.